src/components/footer/index.tsx

```diff
@@ -0,0 +1,90 @@
+"use client";
+
+import React, { useState } from "react";
+import { Box, Section, Text } from "@radix-ui/themes";
+import Image from "next/image";
+import CollapsibleBox from "../boxes/collapsible-box/collapsible-box";
+import {
+  BOX1_ITEMS,
+  BOX2_ITEMS,
+  BOX3_ITEMS,
+} from "../../constants/footer-items-mock";
+import footerLogo from "../../../public/footer-logo.png";
+import Link from "next/link";
+import facebook from "../../../public/facebook.png";
+import teleg from "../../../public/telegram.png";
+import insta from "../../../public/insta.png";
+
+const Footer: React.FC = () => {
+  const [visibleBox, setVisibleBox] = useState<string | null>(null);
```

Comment:
Can we use a better naming? For example `currentFooterSection` or smt like that

Reply:
done